{"uid":"ce92d8952e05f6a1","name":"test_user_can_go_to_cart_page[locked_out_user-secret_sauce]","fullName":"tests.test_TC_016_01_return_to_catalog_after_order.TestSample#test_user_can_go_to_cart_page","historyId":"59f0ca1cf3b0d6c345a24d86a7ed3770","time":{"start":1671195086937,"stop":1671195087229,"duration":292},"description":"Checks whether a user can navigate to the cart page of a website after logging in and adding items to cart.\n\n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        ","descriptionHtml":"<p>Checks whether a user can navigate to the cart page of a website after logging in and adding items to cart.</p>\n<pre><code>    Args:\n        d: WebDriver\n        username: used to specify the username of the user\n        password: used to specify the password of the user\n</code></pre>\n","status":"skipped","statusMessage":"XFAIL\n\nAssertionError: Expected link inventory not found in URL https://www.saucedemo.com/","statusTrace":"self = <SauceDemoProject.tests.test_TC_016_01_return_to_catalog_after_order.TestSample object at 0x7ff0d730dff0>\nd = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed971190ca12131776c4a82a0964a3f0\")>\nusername = 'locked_out_user', password = 'secret_sauce'\n\n    @pytest.mark.parametrize(\n        \"username, password\",\n        [\n            (\"standard_user\", \"secret_sauce\"),\n            (\"performance_glitch_user\", \"secret_sauce\"),\n            (\"problem_user\", \"secret_sauce\"),\n            pytest.param(\n                \"locked_out_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n        ],\n    )\n    def test_user_can_go_to_cart_page(self, d, username, password):\n        \"\"\"Checks whether a user can navigate to the cart page of a website after logging in and adding items to cart.\n    \n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        \"\"\"\n>       self.test_user_can_go_to_products_page(d, username, password)\n\ntests/test_TC_016_01_return_to_catalog_after_order.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_TC_016_01_return_to_catalog_after_order.py:58: in test_user_can_go_to_products_page\n    page.should_be_products_page()\npages/products_page.py:16: in should_be_products_page\n    self.should_be_link(ProductsPageSrc.LINK)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.products_page.ProductsPage object at 0x7ff0d61e3280>\nlink = 'inventory'\n\n    def should_be_link(self, link):\n        \"\"\"Check that the link provided is in the current URL of the browser.\n    \n        Args:\n            link: link browser\n        \"\"\"\n        assert (\n>           link in self.browser.current_url\n        ), f\"Expected link {link} not found in URL {self.browser.current_url}\"\nE       AssertionError: Expected link inventory not found in URL https://www.saucedemo.com/\n\npages/base_page.py:350: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"browser","time":{"start":1671195071115,"stop":1671195071115,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"d","time":{"start":1671195071116,"stop":1671195071980,"duration":864},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"g","time":{"start":1671195071980,"stop":1671195072156,"duration":176},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"Checks whether a user can navigate to the cart page of a website after logging in and adding items to cart.\n\n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        ","status":"skipped","statusMessage":"XFAIL\n\nAssertionError: Expected link inventory not found in URL https://www.saucedemo.com/","statusTrace":"self = <SauceDemoProject.tests.test_TC_016_01_return_to_catalog_after_order.TestSample object at 0x7ff0d730dff0>\nd = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed971190ca12131776c4a82a0964a3f0\")>\nusername = 'locked_out_user', password = 'secret_sauce'\n\n    @pytest.mark.parametrize(\n        \"username, password\",\n        [\n            (\"standard_user\", \"secret_sauce\"),\n            (\"performance_glitch_user\", \"secret_sauce\"),\n            (\"problem_user\", \"secret_sauce\"),\n            pytest.param(\n                \"locked_out_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n        ],\n    )\n    def test_user_can_go_to_cart_page(self, d, username, password):\n        \"\"\"Checks whether a user can navigate to the cart page of a website after logging in and adding items to cart.\n    \n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        \"\"\"\n>       self.test_user_can_go_to_products_page(d, username, password)\n\ntests/test_TC_016_01_return_to_catalog_after_order.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_TC_016_01_return_to_catalog_after_order.py:58: in test_user_can_go_to_products_page\n    page.should_be_products_page()\npages/products_page.py:16: in should_be_products_page\n    self.should_be_link(ProductsPageSrc.LINK)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.products_page.ProductsPage object at 0x7ff0d61e3280>\nlink = 'inventory'\n\n    def should_be_link(self, link):\n        \"\"\"Check that the link provided is in the current URL of the browser.\n    \n        Args:\n            link: link browser\n        \"\"\"\n        assert (\n>           link in self.browser.current_url\n        ), f\"Expected link {link} not found in URL {self.browser.current_url}\"\nE       AssertionError: Expected link inventory not found in URL https://www.saucedemo.com/\n\npages/base_page.py:350: AssertionError","steps":[],"attachments":[{"uid":"4e16079f2f08daeb","name":"Screenshot","source":"4e16079f2f08daeb.png","type":"image/png","size":69588}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"g::0","time":{"start":1671195132958,"stop":1671195133018,"duration":60},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(raises=<class 'AssertionError'>)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_TC_016_01_return_to_catalog_after_order"},{"name":"subSuite","value":"TestSample"},{"name":"host","value":"fv-az360-669"},{"name":"thread","value":"2755-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_TC_016_01_return_to_catalog_after_order"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"password","value":"'secret_sauce'"},{"name":"username","value":"'locked_out_user'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":0,"unknown":0,"total":2},"items":[{"uid":"c8d288a656d10128","reportUrl":"https://Natalia-Tsy.github.io/SauceDemoProject/285//#testresult/c8d288a656d10128","status":"skipped","statusDetails":"XFAIL\n\nAssertionError: Expected link inventory not found in URL https://www.saucedemo.com/","time":{"start":1671195023887,"stop":1671195024192,"duration":305}}]},"tags":["@pytest.mark.xfail(raises=<class 'AssertionError'>)"]},"source":"ce92d8952e05f6a1.json","parameterValues":["'secret_sauce'","'locked_out_user'"]}